ddpui/core/infra_service.py

```
@@ -0,0 +1,110 @@
+        headers=headers,
+        json={"client_name": client_name.replace("_", "-")},
+    )
+    try:
+        response = dalgo_post(
+            f"{INFRA_SERVICE_API_URL}/api/infra/superset",
+            headers=headers,
+            json={"client_name": client_name.replace("_", "-")},
+        )
+    except Exception as e:
+        logger.error(f"Failed to create Superset instance: {str(e)}")
+        raise Exception(f"Failed to create Superset instance: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)